doc/reference/tooling/interactive_console.rst

```diff
@@ -35,35 +34,46 @@ It includes:
-The default is nothing, which means input does not need to end with a delimiter.
-But a common recommendation is to say ``set delimiter ;`` especially if input is SQL.
+The **delimiter** can be changed to any character with ``\set delimiter <character>``.
+By default, the delimiter is empty, which means the input does not need to end
+with a delimiter. To change the delimiter, run ``set delimiter <delimiter>``.
+For example, a common recommendation for SQL input is to use the semicolon delimiter.
+To set it, run ``set delimiter ;``.
 
-The **output format** can be changed to Lua with ``\set output lua``
-or changed to YAML (the default) with ``\set output yaml``.
+The **output format** can be either `YAML <http://yaml.org/spec>`_ (default) or Lua.
+To change the output format, run ``\set output <format>``, for example, ``\set output lua``.
 
-Ordinarily, output from the console has `YAML format <http://yaml.org/spec>`_.
-That means that there is a line for document-start ``"---"``,
-and each item begins on a separate line starting with ``"- "``,
-and each sub-item in a nested structure is indented,
-and there is a line for document-end ``"..."``.
+The default YAML output format is the following:
 
-Optionally, output from the console can have Lua format.
-That means that there are no lines for document-start or document-end,
-and items are not on separate lines (they are only separated by commas),
-and each sub-item in a nested structure is placed inside "``{}``" braces.
-So, when input is a Lua object description, output will equal input.
+*   The output starts from a document-start line ``"---"``.
+*   Each item begins on a separate line starting with ``"- "``.
+*   Each sub-item in a nested structure is indented.
+*   The output ends with a document-end line ``"..."``.
 
-YAML is good for readability.
-Lua is good for re-using results as requests.
-The third format, MsgPack, is good for database storage.
-Currently the default output format is YAML but it may be Lua in a future version,
-and it may be Lua if
-the last :ref:`set_default_output <console-set_default_output>`
-call was ``console.set_default_output('lua')``.
+The alternative Lua format for console output is the following:
+
+*   There are no lines for document-start or document-end.
+*   Items are separated by commas.
+*   Each sub-item in a nested structure is placed inside "``{}``" braces.
+
+So, when an input is a Lua object description, the output in the Lua format equals it.
+
+For the Lua output format, you can specify an **end of statement** symbol.
+It is added to the end of each output statement in the current session and
+can be used for parsing the output by scripts. By default, the end of statement
+symbol is empty. You can change it to any character or character sequence.
+To set an end of statement symbol for the current session, run ``set output lua,local_eos=<symbol>``,
+for example, ``set output lua,local_eos=#``.
```

Comment:
Same as above - get the error without the backslash:
```
tarantool> set output lua,local_eos=#
{error = "[string \"set output lua,local_eos=#\"]:1: \'=\' expected near \'output\'"}
tarantool> \set output lua,local_eos=#
true#
```

And, maybe, it's a matter of preference but I'd place the example in a separate code block to better highlight it. Up to you.